server/src/main/java/org/elasticsearch/rest/action/admin/indices/RestCreateIndexAction.java

```diff
@@ -43,6 +43,9 @@
         LogManager.getLogger(RestCreateIndexAction.class));
     public static final String TYPES_DEPRECATION_MESSAGE = "[types removal] Using include_type_name in create " +
         "index requests is deprecated. The parameter will be removed in the next major version.";
+    public static final String TYPES_DEPRECATION_MESSAGE_6_7_0 = "[types removal] Specifying types in create " +
+        "index requests is deprecated. The parameter include_type_name should be provided and set to false to be " +
+        "compatible with the next major version.";
```

Comment:
In a separate PR I was reconsidering this warning message. The deprecation part is clear enough but the solution only mentions setting a flag "to be compatible with the next major version".  Maybe we should be more explicit stating that the user needs to both:
1) Remove the type from the usual `mapping -> type -> properties` hierarchy and 
2) Tell us they've done this with the `include_type_name=false` param 

It's unfortunate that doing step 1 requires users to also do 2 but that's where we are.

Reply:
That does sound clearer to me! Would you be able to make that update on 6.x? Then I could hold this PR until that merges, and update the message here before merging.